4.analyze_data/notebooks/linear_model/scripts/0.linear_model_concat_coeff.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Perform linear model per CellProfiler feature on concatenated normalized data from plates 5, 3 prime, and 3
+# 
+# We will include 3 co-variates:
+# 
+# 1. Cell count per well contribution
+# 2. Plate contribution
+# 3. Genotype contribution (WT versus Null)
+
+# In[1]:
+
+
+import pathlib
+
+import pandas as pd
+from pycytominer import feature_select
+from pycytominer.cyto_utils import infer_cp_features
+from sklearn.linear_model import LinearRegression
+
+
+# In[2]:
+
+
+# Define inputs and outputs
+data_dir = pathlib.Path("../../../../nf1_cellpainting_data/3.processing_features/data/single_cell_profiles/")
```

Comment:
Does the filepath here reference the directory where the repository is stored (which may vary, depending on how a user has checked out the work)? If so, consider one less `..` to help keep this reference dynamic to the project.

Reply:
Great catch! I have updated this to make sure it is to the exact location in the repo, not sure how I had the extra `../` lol